src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Integration/OpenApiDocumentIntegrationTests.cs

```diff
@@ -42,7 +38,6 @@ public async Task VerifyOpenApiDocument(string documentName, OpenApiSpecVersion
         var outputDirectory = Path.Combine(baseSnapshotsDirectory, version.ToString());
         await Verifier.Verify(json)
             .UseDirectory(outputDirectory)
-            .ScrubLinesWithReplace(line => DateTimeRegex().Replace(line, "[datetime]"))
```

Comment:
Was this that bug I reported last week?

Reply:
Yep! It got fixed in preview9 or preview10 I believe.